sounds/models.py

```diff
@@ -1013,6 +1014,29 @@ class Meta:
         ordering = ("-created",)
 
 
+@receiver(post_delete, sender=Download)
+def update_num_downloads_on_delete(**kwargs):
+    download = kwargs['instance']
+    if download.pack:
```

Comment:
Does django give us the instance object with any related objects loaded? In these two methods we're going to be doing 2 select methods to resolve `pack` and `sound` which will double the number of queries needed for adding a download object (insert, select, select, update) instead of the current insert, update from the trigger.

Can we replace the checks with `if download.pack_id`?

If this can't be done, let's at least put sound first, since it's the most common download object and use `elif` instead of two `if`s